x/microtx/client/rest/query.go

```diff
@@ -0,0 +1,26 @@
+package rest
+
+// Example query handler which delegates the request to the ABCI Query method,
+// allowing queries to fetch bytes directly from the store
+// This method would also need to Unmarshal those bytes and respond in JSON
+// func DataHandler(cliCtx client.Context, storeName string) http.HandlerFunc {
```

Comment:
The rest endpoints only expose the tendermint rpc ABCIQuery, so these make it easier to perform the custom/storename/keyname path queries. I'm leaving this as a comment for an example in the future if we care to implement such a query.